api/src/main/java/org/pmiops/workbench/firecloud/FireCloudServiceImpl.java

```diff
@@ -142,11 +142,11 @@ public FireCloudServiceImpl(
     this.groupsApiProvider = groupsApiProvider;
     this.nihApiProvider = nihApiProvider;
     this.endUserWorkspacesApiProvider = endUserWorkspacesApiProvider;
+    this.endUserLenientTimeoutWorkspacesApiProvider = endUserLenientTimeoutWorkspacesApiProvider;
     this.serviceAccountWorkspaceApiProvider = serviceAccountWorkspaceApiProvider;
     this.statusApiProvider = statusApiProvider;
     this.termsOfServiceApiProvider = termsOfServiceApiProvider;
     this.endUserStaticNotebooksApiProvider = endUserStaticNotebooksApiProvider;
-    this.serviceAccountStaticNotebooksApiProvider = serviceAccountStaticNotebooksApiProvider;
```

Comment:
was this unused?  Should delete `SERVICE_ACCOUNT_STATIC_NOTEBOOKS_API` and all refs to it as well.

Reply:
Yep, unused. Good call - done.